libraries/stdlib/stdlib_defs.mtlx

```diff
@@ -1004,7 +1004,7 @@
     <input name="type" type="integer" uiname="Noise Type" uifolder="Common" uisoftmin="0" uisoftmax="3" value="0" enum="Perlin,Cell,Worley,Fractal" enumvalues="0,1,2,3" doc="Menu to select the type of noise: Perlin, Cell, Worley, or Fractal. Default is Perlin." />
     <output name="out" type="float" />
   </nodedef>
-  
+
```

Comment:
Indented empty lines are not preserved by design as spaces and tabs are not preserved.